rel-eng/tito.props

```diff
@@ -387,58 +389,36 @@ whitelist = nodejs-babel-plugin-module-resolver
   rubygem-httparty
   rubygem-http-form_data
   rubygem-http_parser.rb
+  rubygem-infoblox
   rubygem-jgrep
   rubygem-jquery-matchheight-rails
   rubygem-kubeclient
+  rubygem-logify
   rubygem-ms_rest
   rubygem-ms_rest_azure
   rubygem-multi_xml
   rubygem-net-ssh-gateway
   rubygem-net-ssh-krb
   rubygem-net-ssh-multi
+  rubygem-openscap
+  rubygem-openscap_parser
   rubygem-ovirt_provision_plugin
   rubygem-parse-cron
   rubygem-pdf-core
   rubygem-polyglot
   rubygem-prawn
   rubygem-puppetdb_foreman
+  rubygem-radcli
   rubygem-rainbow
   rubygem-recursive-open-struct
+  rubygem-route53
   rubygem-rprogram
   rubygem-rqrcode
+  rubygem-ruby-hmac
   rubygem-ruby-nmap
   rubygem-safe_yaml
   rubygem-smart_proxy_dynflow_core
   rubygem-smart_proxy_salt_core
-  rubygem-systemu
-  rubygem-timeliness
-  rubygem-ttfunk
-  rubygem-vault
-  rubygem-wicked
-  rubygem-zscheduler
-
-[foreman-plugins-nightly-nonscl-rhel7]
-disttag = .el7
-whitelist = ansiblerole-foreman_scap_client
-  ansiblerole-insights-client
-  foreman-discovery-image-service
-  puppet-foreman_scap_client
-  rubygem-chef-api
-  rubygem-faraday
-  rubygem-faraday_middleware
-  rubygem-fast_gettext
-  rubygem-infoblox
-  rubygem-logify
-  rubygem-mysql2
```

Comment:
If you're dropping this, should we also drop the package altogether so it's not orphaned?

Reply:
It's not actually dropped. Just dropped from this tag.